src/application/runner.go

```diff
@@ -38,7 +39,8 @@ func NewRunner(appContext types.AppContext, writer io.Writer, homeDir, dockerCom
 		HomeDir:                  homeDir,
 		ServiceConfigs:           serviceConfigs,
 		BuiltDependencies:        allBuiltDependencies,
-		DockerComposeDir:         path.Join(appContext.Location, "tmp"),
+		DockerComposeDir:         path.Join(appContext.Location, "docker-compose"),
+		DockerComposeFileName:    buildMode.GetDockerComposeFileName(),
```

Comment:
I don't thin k we need to save this on the runner and it can be computed in `Run`